core/templates/pages/exploration-editor-page/translation-tab/services/translation-tab-active-mode.service.ts

```diff
@@ -14,28 +14,37 @@
 
 /**
  * @fileoverview Service to change and validate active mode in the translation
- * tab.
+ *      tab.
  */
 
-require(
-  'pages/exploration-editor-page/exploration-editor-page.constants.ajs.ts');
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
 
-angular.module('oppia').factory('TranslationTabActiveModeService', [
-  'TRANSLATION_MODE', 'VOICEOVER_MODE',
-  function(TRANSLATION_MODE, VOICEOVER_MODE) {
-    var activeMode = null;
-    return {
-      activateVoiceoverMode: function() {
-        activeMode = VOICEOVER_MODE;
-      },
-      activateTranslationMode: function() {
-        activeMode = TRANSLATION_MODE;
-      },
-      isTranslationModeActive: function() {
-        return activeMode === TRANSLATION_MODE;
-      },
-      isVoiceoverModeActive: function() {
-        return activeMode === VOICEOVER_MODE;
-      }
-    };
-  }]);
+import { ExplorationEditorPageConstants } from
+  'pages/exploration-editor-page/exploration-editor-page.constants';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class TranslationTabActiveModeService {
+  constructor() {}
```

Comment:
Remove this if we aren't doing anything in the `constructor`. (The default `constructor` is the same as this one)